samples/mpsl/timeslot/CMakeLists.txt

```diff
@@ -8,6 +8,11 @@ cmake_minimum_required(VERSION 3.8)
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(NONE)
 
+if ((BOARD STREQUAL nrf5340pdk_nrf5340_cpuappns) OR
```

Comment:
For later: should this check be on SOC rather than BOARD? Customers typically specify their own boards.

Reply:
Good point. I just used the same pattern as in `samples/CMakeLists.txt`. We should replace it there as well.

Reply:
Here you go :) https://github.com/nrfconnect/sdk-nrf/pull/2476